integration/tests/cook/test_cli.py

```diff
@@ -871,7 +871,9 @@ def entry(name):
     def test_show_progress_message(self):
         executor = util.get_job_executor_type(self.cook_url)
         cp, uuids = cli.submit('echo "progress: 99 We\'re so close!"', self.cook_url,
-                               submit_flags=f'--executor {executor}')
+                               submit_flags=f'--executor {executor} '
+                                            '--env EXECUTOR_PROGRESS_OUTPUT_FILE_ENV=PROGRESS_OUTPUT_FILE '
+                                            '--env PROGRESS_OUTPUT_FILE=stdout')
```

Comment:
It's unfortunate that this test (and other similar ones) are getting more complex simply to post progress. We discussed offline the possibility of making the default progress output file configurable, so that in our test and local environments we can configure it to `stdout`.